wurst/data/LinkedList.wurst

```diff
@@ -327,6 +329,43 @@ public interface LLItrClosure<T>
 public interface LinkedListUpdater<T>
 	function update(T t) returns T
 
+/*  String processing extra functions 
+	The only reason this is here is that it can't be in String.wurst due to dependency problems */
+
+/** Splits the string by another string 
+	"this..is..an..example" split by the string ".." splits into a list containing four strings ["this", "is", "an", "example"] */
+public function string.split(string splitBy) returns LinkedList<string>
+	let list = new LinkedList<string>
+	let offset = splitBy.length()
+	var str = this
+	while true
```

Comment:
`while str.indexOf(splitBy) != -1`

Reply:
I don't want to call indexOf twice, which is why I did this